runtimes/main/src/pallets_governance.rs

```diff
@@ -189,3 +190,42 @@ impl pallet_amendments::Config for Runtime {
     type PalletsOrigin = OriginCaller;
     type WeightInfo = pallet_amendments::weights::SubstrateWeight<Runtime>;
 }
+
+parameter_types! {
+    pub const MaxWnodlOracles: u32 = u32::MAX;
+    pub const MaxKnownCustomers: u32 = u32::MAX;
+}
+
+impl pallet_membership::Config<pallet_membership::Instance6> for Runtime {
+    type Event = Event;
+    type AddOrigin = pallet_collective::EnsureMembers<_2, AccountId, TechnicalCollective>;
+    type RemoveOrigin = pallet_collective::EnsureMembers<_2, AccountId, TechnicalCollective>;
+    type SwapOrigin = pallet_collective::EnsureMembers<_2, AccountId, TechnicalCollective>;
+    type ResetOrigin = pallet_collective::EnsureMembers<_2, AccountId, TechnicalCollective>;
+    type PrimeOrigin = pallet_collective::EnsureMembers<_2, AccountId, TechnicalCollective>;
+    type MembershipInitialized = ();
+    type MembershipChanged = ();
+    type MaxMembers = MaxWnodlOracles;
+    type WeightInfo = pallet_membership::weights::SubstrateWeight<Runtime>;
+}
+
+impl pallet_membership::Config<pallet_membership::Instance7> for Runtime {
+    type Event = Event;
+    type AddOrigin = pallet_collective::EnsureMember<AccountId, FinancialCollective>;
+    type RemoveOrigin = pallet_collective::EnsureMember<AccountId, FinancialCollective>;
+    type SwapOrigin = pallet_collective::EnsureMember<AccountId, FinancialCollective>;
+    type ResetOrigin = pallet_collective::EnsureMember<AccountId, FinancialCollective>;
+    type PrimeOrigin = pallet_collective::EnsureMember<AccountId, FinancialCollective>;
+    type MembershipInitialized = ();
+    type MembershipChanged = ();
+    type MaxMembers = MaxKnownCustomers;
+    type WeightInfo = pallet_membership::weights::SubstrateWeight<Runtime>;
+}
```

Comment:
```suggestion
impl pallet_membership::Config<pallet_membership::Instance6> for Runtime {
    type Event = Event;
    type AddOrigin = pallet_collective::EnsureProportionMoreThan<_1, _2, AccountId, TechnicalCollective>;
    type RemoveOrigin = pallet_collective::EnsureProportionMoreThan<_1, _2, AccountId, TechnicalCollective>;
    type SwapOrigin = pallet_collective::EnsureProportionMoreThan<_1, _2, AccountId, TechnicalCollective>;
    type ResetOrigin = pallet_collective::EnsureProportionMoreThan<_1, _2, AccountId, TechnicalCollective>;
    type PrimeOrigin = pallet_collective::EnsureProportionMoreThan<_1, _2, AccountId, TechnicalCollective>;
    type MembershipInitialized = ();
    type MembershipChanged = ();
    type MaxMembers = MaxWnodlOracles;
    type WeightInfo = pallet_membership::weights::SubstrateWeight<Runtime>;
}

impl pallet_membership::Config<pallet_membership::Instance7> for Runtime {
    type Event = Event;
    type AddOrigin = pallet_collective::EnsureProportionMoreThan<_1, _2, AccountId, FinancialCollective>;
    type RemoveOrigin = pallet_collective::EnsureProportionMoreThan<_1, _2, AccountId, FinancialCollective>;
    type SwapOrigin = pallet_collective::EnsureProportionMoreThan<_1, _2, AccountId, FinancialCollective>;
    type ResetOrigin = pallet_collective::EnsureProportionMoreThan<_1, _2, AccountId, FinancialCollective>;
    type PrimeOrigin = pallet_collective::EnsureProportionMoreThan<_1, _2, AccountId, FinancialCollective>;
    type MembershipInitialized = ();
    type MembershipChanged = ();
    type MaxMembers = MaxKnownCustomers;
    type WeightInfo = pallet_membership::weights::SubstrateWeight<Runtime>;
}
```

Reply:
For sure for adding Oracles. Just a question about adding known customers, wouldn't it make it very hard if we require more than half of the committee to vote per every customer? Or perhaps we are thinking of batch adding?

Reply:
We will do batches :)